avoidance/QuadCopterStopAvoidance.cc

```diff
@@ -55,6 +55,11 @@ void QuadCopterStopAvoidance::avoid(const std::vector<Obstacle> &detection)
 
     // If no obstacle was detected, do nothing.
     if (detection.size() == 0) {
+        if (this->vehicle->mav->is_brake_active()) {
+            this->vehicle->mav->brake(true);
```

Comment:
Agree! Actually this is here by mistake. I'm using this to test on a real drone but I wasn't thinking in merging it yet :-). I will leave this one out for now.